### DISCUSSION TITLE
every term must have a definition

### DISCUSSION THREAD

**Discussion by VladimirAlexiev** [eprocurementontology/eprocurementontology#520] [Ideas]
This query shows 194 terms without a definition:
```sparql
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
select * {
    ?x rdfs:isDefinedBy  <http://data.europa.eu/a4g/ontology>
    filter not exists {?x rdfs:comment ?def}
} order by ?x
```

19 of them are classes and the rest are properties.

A prop like this is hard to understand, and using the IT jargon word "Indicator" doesn't make it any easier:
```ttl
epo:hasVATIncludedIndicator
        rdfs:label        "V a t included indicator"@en .
```
See how much easier is to understand this:
```ttl
epo:isVATIncluded
        rdfs:label        "Is VAT included"@en ;
        rdfs:comment      "Whether Value Added Tax (VAT) is included in the value amount(s)".
```

**Comment by VladimirAlexiev** [comment]
Another example: What's the difference between these classes?

```ttl
epo:Amount  a             owl:Class ;
        rdfs:isDefinedBy  <http://data.europa.eu/a4g/ontology> ;
        rdfs:label        "Amount"@en ;
        skos:prefLabel    "Amount"@en .
epo:Value  a              owl:Class ;
        rdfs:isDefinedBy  <http://data.europa.eu/a4g/ontology> ;
        rdfs:label        "Value"@en ;
        skos:prefLabel    "Value"@en .
```
From examination I see:
```ttl
epo:hasMinimumAmount  rdfs:domain  epo:Value ; rdfs:range   epo:Amount .
```
and that
- Amount has epo:hasAmountValue, epo:hasCurrencyID
- Value has epo:hasVATIncludedIndicator, epo:hasMinimumAmount, epo:hasMaximumAmount, epo:hasOverallAmount, etc

So 
- epo:Amount: "A number and currency code"
- epo:Value: "Particulars describing a tendering amount, including minimum, maximum, whether VAT is included, overall, etc"

**Comment by giorgialodi** [comment]
@VladimirAlexiev Value is currently under revision by the working group with a proposal to use MonetaryValue  instead of Value and to use just this class without the need of the class Amount.  In any case, in version 2.0.1, Amount is a general amount concept with a value and a currency. Value is a Procurement Value,  more domain specific with maximum and minimum value and VAT.
We need to complete the definitions of each term.

**Comment by VladimirAlexiev** [comment]
@andreea-pasare @giorgialodi can you run the query on a current version of the ontologies to see how many lack definition?
